spring-cloud-dataflow-docs/src/main/asciidoc/configuration-local.adoc

```diff
@@ -581,35 +581,12 @@ uaac user get <username>
 
 For local testing and development, you may also use the Resource and Authorization
 Server support provided by
-https://projects.spring.io/spring-security-oauth/[Spring Security OAuth]. It
-allows you to easily create your own (very basic) OAuth2 Server with the following simple annotations:
+https://spring.io/projects/spring-security/[Spring Security]. It
+allows you to easily create your own OAuth2 Server by configuring the SecurityFilterChain.
 
-* `@EnableResourceServer`
-* `@EnableAuthorizationServer`
+Samples can be found at:
+https://docs.spring.io/spring-security/reference/samples.html[Spring Security Samples]
 
-NOTE: In fact the UAA uses Spring Security OAuth2 under the covers, thus the basic endpoints
```

Comment:
We should check the other docs to make sure this is not in them as well. 
- [ ] skipper ref docs
- [ ] scdf ref docs (that is actually this doc here)
- [ ] dataflow.spring.io (I doubt it's in there but we should check)
- [ ] scdf samples (this doc is no longer published outward so no need to check there)

Reply:
I searched the docs and only one referenced the deprecated library. I changed the docs to refer to latest Spring Security and Samples.